src/pages/Login/index.tsx

```diff
@@ -7,17 +7,37 @@ import TextInputComponent from '../../components/TextInput';
 import TextInputEnum from '../../enums/TextInput.enum';
 import DivComponent from '../../components/DivContainer';
 import TextLabel from '../../components/TextLabel';
+import {LoginDTO} from '../../types/User.type';
+import {useUserCredentials} from '../../hooks/useUserHooks';
 import DividerComponent from '../../components/Divider';
+import {Formik} from 'formik';
 
 export default function Login(props: any) {
+  const {sendLoginQRUser} = useUserCredentials();
+
+  const initValues: LoginDTO = {
+    email: '',
+    password: '',
+  };
+
   const {navigation} = props;
 
   const onSignup = () => {
     navigation.navigate('Register');
   };
 
-  const onGoToDashboard = () => {
-    navigation.navigate('Dashboard');
+  const onLoginUser = (values: LoginDTO) => {
+    sendLoginQRUser(values)
+      .then((response: any) => {
+        if (response.length === 0) {
```

Comment:
i think you should check it this way if(!response) since the response of login is and object.

Reply:
mao ni implementation ron, sa second commits @chappy11